firestore-send-email/extension.yaml

```diff
@@ -181,6 +181,28 @@ params:
       if the template is specified in the added email document.
     type: string
 
+  - param: FIRESTORE_EXPIRE_AT
```

Comment:
I'm not sure if this is the best name for this param, any suggestions @dackers86 ?